packages/ui/src/ThemeProvider/FontFaces.tsx

```diff
@@ -0,0 +1,116 @@
+import { createGlobalStyle } from 'styled-components';
+
+import poppinsItalic from './fonts/Poppins-Italic.woff2?url';
+import poppinsItalicLatinExt from './fonts/Poppins-Italic-LatinExt.woff2?url';
+import poppinsMedium from './fonts/Poppins-Medium.woff2?url';
+import poppinsMediumItalic from './fonts/Poppins-MediumItalic.woff2?url';
+import poppinsMediumItalicLatinExt from './fonts/Poppins-MediumItalic-LatinExt.woff2?url';
+import poppinsMediumLatinExt from './fonts/Poppins-Medium-LatinExt.woff2?url';
+import poppinsRegular from './fonts/Poppins-Regular.woff2?url';
+import poppinsRegularLatinExt from './fonts/Poppins-Regular-LatinExt.woff2?url';
+import workSansMedium from './fonts/WorkSans-Medium.woff2?url';
+import workSansMediumLatinExt from './fonts/WorkSans-Medium-LatinExt.woff2?url';
+import workSansMediumVietnamese from './fonts/WorkSans-Medium-Vietnamese.woff2?url';
```

Comment:
Downloaded these font files individually from Google Fonts, since Google Fonts somehow has SUPER compressed versions of them.